src/applications/caregivers/components/NeedHelpFooter/index.jsx

```diff
@@ -10,34 +11,41 @@ const NeedHelpFooter = () => {
       <span>Need help filling out this form? Call our toll-free number: </span>
 
       <section className={sectionClassNames}>
-        <a href="tel:18772228387">1-877-222-8387</a>
+        <a href={links.VAHelpLine.label}>{links.VAHelpLine.link}</a>
         <span>
-          TTY: <a href="tel:18008778339">1-800-877-8339</a>
+          TTY:{' '}
+          <a href={links.VAHelpLine.linkTTL}>{links.VAHelpLine.labelTTL}</a>
         </span>
         <span>Monday through Friday, 8:00a.m. — 8:00p.m. ET.</span>
```

Comment:
This might be a candidate for {links.VAHelpLine.availability}

Reply:
The time should include a "to" instead of a dash for screen readers: `8:00 a.m. to 8:00 p.m. ET.`